environments/production/postgresql.yml

```diff
@@ -148,3 +166,5 @@ dbs:
           - pgbouncer7
           - pgbouncer8
           - pgbouncer9
+          - pgbouncer10
+          - pgbouncer11
```

Comment:
Confirm that these added to all DB's not for the sake of the rollout? And they will be removed from most DBs in stage 2?

Reply:
This seems to be our convention, which allows them to access all dbs, but does not affect the routing. All other pgbouncers are listed under every DB, so I tried to keep that consistent.